editor/writing-flow/index.js

```diff
@@ -71,7 +103,9 @@ class WritingFlow extends Component {
 		return (
 			<div
 				ref={ this.bindContainer }
-				onKeyDown={ this.onKeyDown }>
+				onKeyDown={ this.onKeyDown }
+				onMouseDown={ () => this.verticalRect = null }
```

Comment:
It would have been good to assign this to a bound instance method so that it's not creating a new function reference (and thereby applying a change diff) on every render.

Reply:
Darn, I'm usually stringent about those.